requirements.tools.in

```diff
@@ -0,0 +1,3 @@
+# requirements for tools we want in prod, but not required
+litecli==1.5.0
+py-spy==0.3.4
```

Comment:
`py-spy` is a binary dep, does this definitely work with our vendoring-for-windows process?